src/main/java/com/ultimatelogger/multiplatform/output/LoggingIfIsOnMultiPriorityLogger.kt

```diff
@@ -190,5 +190,19 @@ open class LoggingIfIsOnMultiPriorityLogger : UltimateLogger {
         }
     }
 
+    override fun <AnyT> todo(anything: AnyT?,
+                    withThreadName: Boolean?,
+                    withFileNameAndLineNum: Boolean?,
+                    withClassName: Boolean?,
+                    withMethodName: Boolean?) {
+        runIfIsLoggingOn {
```

Comment:
Didn't notice before, but the proper name in English would be `runIfLogginIsOn`.